packages/iframe-coordinator/src/messages/PromptOnLeave.ts

```diff
@@ -0,0 +1,30 @@
+import { boolean, constant, Decoder, object } from 'decoders';
+import { labeledDecoder, LabeledMsg } from './LabeledMsg';
+
+/** A prompt on leave notification to be displayed
+ * by the host application
```

Comment:
I'd add some more detail to this to clarify how the message is intended to impact navigation. These comments are used in the [generated documentation](https://apps.inindca.com/common-ui-docs/iframe-coordinator/interfaces/client.NavRequest.html)